projects/control-service/projects/pipelines_control_service/src/integration-test/resources/application-test.properties

```diff
@@ -12,20 +12,26 @@ datajobs.aws.secretAccessKey=
 datajobs.vdk_options_ini=
 datajobs.vdk.image=registry.hub.docker.com/versatiledatakit/quickstart-vdk:release
 
-datajobs.deployment.k8s.kubeconfig=${DEPLOYMENT_K8S_KUBECONFIG}
-datajobs.deployment.k8s.namespace=${DEPLOYMENT_K8S_NAMESPACE}
+datajobs.deployment.k8s.kubeconfig=${DEPLOYMENT_K8S_KUBECONFIG:}
+datajobs.deployment.k8s.namespace=${DEPLOYMENT_K8S_NAMESPACE:default}
 datajobs.control.k8s.k8sSupportsV1CronJob=false
 
-datajobs.control.k8s.kubeconfig=${CONTROL_K8S_KUBECONFIG}
-datajobs.control.k8s.namespace=${CONTROL_K8S_NAMESPACE}
+datajobs.control.k8s.kubeconfig=${CONTROL_K8S_KUBECONFIG:}
+datajobs.control.k8s.namespace=${CONTROL_K8S_NAMESPACE:default}
```

Comment:
is default actually used in case empty default provided? why adding `default` explicitly?

Reply:
less is more, deleting unused code is great; if default not explicitly needed, then why including it